src/net_processing.cpp

```diff
@@ -2351,7 +2351,7 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
 
         // Be shy and don't send version until we hear
         if (pfrom.IsInboundConn())
```

Comment:
style-nit in commit 3b81ec59d15c17bb1c39eeb044c113117c41071d:

Could add `{`

Reply:
I joined the lines since the if block is one line.